docs/content/how-tos/rule-configuration.md

```diff
@@ -114,4 +114,5 @@ The following rules can be specified for linting.
 - [CyclomaticComplexity (FL0071)](rules/FL0071.html)
 - [FailwithBadUsage (FL0072)](rules/FL0072.html)
 - [FavourReRaise (FL0073)](rules/FL0073.html)
-- [FavourConsistentThis (FL0074)](rules/FL0074.html)
```

Comment:
about the commit message: the character * is used when writing a list of things, but here there is only one line, not many, so why do you use the * character? there's no need for it